packages/api/test/integration.test.ts

```diff
@@ -179,28 +144,19 @@ describe('integration tests', function () {
         expect(res.requestBody.split(`--${res.boundary}`).filter(Boolean)).to.deep.equal([
           '\r\nContent-Disposition: form-data; name="orderId"\r\n\r\n1234\r\n',
           '\r\nContent-Disposition: form-data; name="userId"\r\n\r\n5678\r\n',
-          '\r\nContent-Disposition: form-data; name="documentFile"; filename="hello.txt"\r\nContent-Type: text/plain\r\n\r\nHello world!\n\r\n',
+          isNode18
+            ? '\r\nContent-Disposition: form-data; name="documentFile"; filename="hello.txt"\r\nContent-Type: application/octet-stream\r\n\r\nHello world!\n\r\n'
+            : '\r\nContent-Disposition: form-data; name="documentFile"; filename="hello.txt"\r\nContent-Type: text/plain\r\n\r\nHello world!\n\r\n',
           '--\r\n\r\n',
         ]);
 
-        expect(res.headers).to.have.deep.property('content-type', [`multipart/form-data; boundary=${res.boundary}`]);
-        expect(res.headers).to.have.deep.property('content-length', ['389']);
+        expect(res.headers).to.have.deep.property('content-type', `multipart/form-data; boundary=${res.boundary}`);
+        expect(res.headers).to.have.deep.property('content-length', isNode18 ? '403' : '389');
```

Comment:
Tests with Node 18 are funky and I had this same problem when adding Node 18 support to [fetch-har](https://github.com/readmeio/fetch-har/pull/265#discussion_r860366108). 🤷